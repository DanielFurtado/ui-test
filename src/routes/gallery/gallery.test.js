import ImageGallery from '../../components/image-gallery/image-gallery.component';
import renderer from 'react-test-renderer';

it('renders', () => {
  const component = renderer.create(<ImageGallery />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});