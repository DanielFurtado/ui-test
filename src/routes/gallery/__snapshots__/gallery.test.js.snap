// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="image-gallery-container"
>
  <div
    className="image-gallery-heading"
  >
    <h1>
      Photo gallery
    </h1>
    <p>
      Some of our clients have shared some of their amazing photographic work with us here at Motorway! This is truly inspirational!
    </p>
  </div>
  <div
    className="image-gallery-wrapper"
  >
    <ul
      aria-label="Pagination"
      className="pagination"
      role="navigation"
    >
      <li
        className="page-item disabled"
      >
        <a
          aria-disabled="true"
          aria-label="Previous page"
          className="page-link "
          onClick={[Function]}
          onKeyPress={[Function]}
          rel="prev"
          role="button"
          tabIndex="-1"
        >
          &lt;
        </a>
      </li>
      <li
        className="page-item"
      >
        <a
          aria-disabled="false"
          aria-label="Next page"
          className="page-link"
          onClick={[Function]}
          onKeyPress={[Function]}
          rel="next"
          role="button"
          tabIndex="0"
        >
          &gt;
        </a>
      </li>
    </ul>
  </div>
</div>
`;
